packages/dds/map/src/mapKernel.ts

```diff
@@ -499,6 +501,16 @@ export class MapKernel implements IValueTypeCreator {
         return false;
     }
 
+    public tryRebaseOp(op: any, localOpMetadata: unknown): boolean {
+        const type: string = op.type;
+        if (this.messageHandlers.has(type)) {
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            this.messageHandlers.get(type)!.rebaseOp(op as IMapOperation, localOpMetadata);
+            return true;
+        }
+        return false;
```

Comment:
Not very clear what should happen on a false path - the only caller does not check result. So it feels like it either needs to be void function, or either caller or callee should do something (log error?) when we take this branch.